src/main/java/servlets/module/challenge/CsrfChallengeTargetJSON.java

```diff
@@ -61,7 +60,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     ResourceBundle csrfGenerics =
         ResourceBundle.getBundle("i18n.servlets.challenges.csrf.csrfGenerics", locale);
 
-    response.setContentType("text/html");

```

Comment:
Should we not be setting the content type?

Reply:
You are right, actually there is no reason to remove this.